ush/python/pygfs/task/atm_analysis.py

```diff
@@ -143,6 +144,21 @@ def initialize_analysis(self) -> None:
         FileHandler(bias_dict).sync()
         logger.debug(f"Bias correction files:\n{pformat(bias_dict)}")
 
+        # Extract radiance bias correction files from tarball
+        for action, filelist in bias_dict.items():
+            if 'copy' in action:
+                for sublist in filelist:
+                    if len(sublist) != 2:
+                        raise Exception(
+                            f"List must be of the form ['src', 'dest'], not {sublist}")
+                    src = sublist[0]
+                    if re.search(".tar", src):
+                        radtar = src
+                        with tarfile.open(radtar, "r") as radbcor:
+                            radbcor.extractall(path=os.path.join(self.task_config.DATA, 'obs'))
+                            logger.info(f"Extract {radbcor.getnames()}")
+                        radbcor.close()
+
```

Comment:
I see that this identical block of code is also in atmens_analysis.py. Can this be rolled into ```Jedi.get_bias_dict()``` or made into another method of ```Jedi```?

Reply:
Sure, I'm all for not replicating code.  Is it OK to invoke `FileHandler` from inside `jedi.py`?

Reply:
I assume it's fine

Reply:
In the current setup we pass the dictionary returned by `get_bias_dict` to `FileHandler`.  This syncs the radiance tarball to the run directory.  Then the block you quotes extracts _satelllite_sensor_ specific bias correction files from the tarball.  

The new `get_bias` in `jedi.py` needs to sync the tarball to the run directory and then untar it.  We are moving the sync and untar operations from outside to inside `jedi.py`.  Once this is done, `atm_analysis.py` and `atmens_analysis.py` invoke `get_bias` from `jedi.py` to populate the run directory with radiance bias correction files.

Reply:
@DavidNew-NOAA , your good suggestion has been implemented.  Done at 73e56a7f.   All `test_gdasapp` _Passed_ with this GDASApp hash.